src/editor/components/TextureContainerEditor.js

```diff
@@ -8,9 +8,10 @@ export default class TextureContainerEditor extends ContainerEditor {
 
   _renderNode($$, node) {
     let api = this.context.api
+    let disabled = this.props.disabled
     if (!node) throw new Error("'node' is mandatory")
 
-    let props = { node }
+    let props = { node, disabled }
```

Comment:
I will fix this in substance land

Reply:
If you wait some minutes I can push a better version.

Reply:
please push :)

Reply:
Yeah.. I am about to do that. Just our yesterdays work needs some 'polishing' ;)

Reply:
I'm working on #564 right now... Take your time.